rabbitmq/meta/heka.yml

```diff
@@ -1,15 +1,53 @@
-{%- if pillar.rabbitmq.server is defined %}
-input:
-  logstreamer_rabbitmq:
-	type: "LogstreamerInput"
-	log_directory: "/var/log/rabbitmq"
-	file_match: 'rabbit@.+\.log\.?(?P<Index>\d+)?(.gz)?'
-	priority: ["^Index"]
-	decoder: "rabbitmq"
-	oldest_duration: "168h"
-decoder:
-  rabbitmq:
-    type: "SandboxDecoder"
-    filename: "lua_modules/decoders/rabbitmq.lua"
-    module_directory: "/usr/share/heka/lua_modules;/usr/share/heka/lua_modules/common"
-{% endif %}
+log_collector:
+  decoder:
+    rabbitmq_service:
+      engine: sandbox
+      module_file: /usr/share/lma_collector/decoders/rabbitmq.lua
+      module_dir: /usr/share/lma_collector_modules;/usr/share/heka/lua_modules
+      config:
+        include_full_notification: false
+  input:
+    rabbitmq_log_stream:
+      engine: logstreamer
+      log_directory: "/var/log/rabbitmq"
+      file_match: 'rabbit@(?P<Node>.+)\.log$'
+      differentiator: ["rabbitmq.", "Node"]
+      decoder: "rabbitmq_decoder"
+      splitter: "rabbitmq_splitter"
+  splitter:
+    rabbitmq_splitter:
+      engine: token
+      delimiter: "\n\n(=[^=]+====)"
+      delimiter_eol: false
+metric_collector:
+  filter:
```

Comment:
We have a requirement as mentioned by @elemoine: the operator must be able to add/disable/override alarm definitions in the salt-model repo. 

@cznewt I don't think having alarm definitions in <service>/meta/*yml allows the operator to override alarm definitions ?

The heka formulas must consume pillar data (reclass parameters) to generate heka sandbox filters (so called AFD filters + alarm definitions in LUA) for services.
We must follow in someway the actual heka formulas shipping its own heka plugins [0] [1] but declared/classified by other services.
[0] https://github.com/tcpcloud/salt-formula-heka/blob/master/heka/server.sls#L184-L199
[1] https://github.com/tcpcloud/salt-formula-heka/blob/master/metadata/service/server/elastic/shipper.yml